### PR TITLE
Add missing DIRAC dependencies

### PR DESCRIPTION
**Pull request by chrisburr**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by chrisburr** [comment]
@conda-forge/staged-recipes This is ready for review.

The status isn’t being reported correctly but you can check the builds [here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=32249).

File: recipes/caniusepython3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - backports.functools_lru_cache
+    - distlib
+    - futures  # [py2k]
```
Comment (beckermr) [inline recipes/caniusepython3/meta.yaml]: There is a futures package for py3, so this selector can be removed and then the whole recipe made noarch I think.

Comment (chrisburr) [reply recipes/caniusepython3/meta.yaml]: I had it like that originally but `futures` only has Python 2.7 binaries:
https://anaconda.org/conda-forge/futures/files

Comment (beckermr) [reply recipes/caniusepython3/meta.yaml]: ohhhh shit you are right. 🤦‍♂

Comment (beckermr) [reply recipes/caniusepython3/meta.yaml]: I read "future" instead of "futures". Grrrrr.

**Comment by chrisburr** [comment]
Thanks @beckermr! 🍰

**Comment by beckermr** [comment]
mmmmm...now I want cake...

**Pull request merged**